ui.R

```diff
@@ -0,0 +1,106 @@
+library(dplyr)
+
+# get list of clusters from folder
+all_clusters = tibble::as_tibble(
+  list.files(
+    glue::glue("www/data/wcdemo/scanner_output")
+  )) %>%
+  filter(stringr::str_detect(value, pattern = "\\.", negate = TRUE)) %>%
+  pull(value)
+
+ui = shiny::navbarPage(
+  # title
+  title = "tfpbrowser",
+
+  # theme
+  theme = bslib::bs_theme(version = 4,
+                          bootswatch = "minty",
+                          bg = "#EBEEEE",
+                          fg = "#002147",
+                          primary = "#003E74",
+                          secondary = "#9D9D9D"),
+
+  # Input widgets
+  shiny::tabPanel(
+    title = "Data",
+    shiny::fluidRow(
+      # left hand side - show tree (static html output from tfpscanner)
+      shiny::column(6,
+        # show treeview widget
+        shiny::wellPanel(
+          shiny::fluidRow(shiny::column(
+            6,
```

Comment:
Is there a reason why you have another fluid row inside the well panel? It looks like you've got some style css that sets a manual width/height of the treeview iframe (L9-10 in server.R) to fill the entire well anyway.

Also that manual width/height in the server means that if you resize the window the iframe overlaps with all the inputs on the right.

Not sure if this is too nitpicky at this stage and bound to change when you refactor/develop further but thought I'd point it out! 😇

Reply:
I've removed the extra fluid row, and have adjusted iframe width to 100% to fill the panel instead